instructions/FOSS_repo_list.pl

```diff
@@ -17,7 +17,7 @@
    { name => "zm-clam-scanner-store",                },
    { name => "zm-clientuploader-admin-zimlet",       },
    { name => "zm-clientuploader-store",              },
-   { name => "zm-core-utils",                        },
+   { name => "zm-core-utils",                       branch => "feature/zms-38" },
```

Comment:
Gren I've got the `FOSS_repo_list` updates isolated in separate commits that will be rebased out once everything is approved.  There is note about that in the PR description. I didn't want to rebase them out until everything else is correct as it allows me to create installs for testing with all the necessary code.